xml/System.Security.Cryptography/RSAOpenSsl.xml

```diff
@@ -475,9 +475,9 @@ The decryption operation failed.</exception>
         <summary>Gets a <see cref="T:System.Security.Cryptography.SafeEvpPKeyHandle" /> representation of the cryptographic key.</summary>
         <returns>A <see cref="T:System.Security.Cryptography.SafeEvpPKeyHandle" /> representation of the cryptographic key.</returns>
         <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
-## Remarks  
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
```

Comment:
the sentence below is semi-correct: it's true the lifetime will be independent and disposing is safe but modifying operations (i.e. direct calls to modifying OpenSSL APIs) on the new handle will use the same underlying object - the new instance is simply an upref